guide/english/computer-hardware/cooling/index.md

```diff
@@ -5,31 +5,31 @@ title: Cooling Systems
 
 ## Environmental Considerations
 
 * Keep it clean - Dust can clog up fans, heat sinks can act as an insulating layer trapping heat. Use compressed air to clean dust from the internal components. Maintenance of liquid cooling systems should be considered as well. Many forms of liquid cooling require regular flushing to keep the tubes clean.
 * Make sure to keep the room clean to keep dust and hair out of the fans and cooling systems to keep them functioning properly.
-* Case Covers - Contrary to what you read on the internet, keeping the case covers on the computer will aid cooling. Absence of the case covers disrupts air flow and can actually increase the heat inside the case.
+* Case Covers - Contrary to what you read on the internet, keeping the case covers on the computer will aid cooling. The absence of the case covers disrupts air flow and can actually increase the heat inside the case.
 * Room ventilation - Place the computer in a neutral location away from the rooms environmental registers. Keep the computer away from room heaters or vents at all times.
 * Room placement - Keep the computer away from windows or areas of strong sunlight exposure. Placement in closets can also provide challenges to cooling. The case should be located in a way that allows good airflow. It is recommended to keep the computer away from any carpet-like material, which can easily bring dust into the computer.
 * Space around the computer - Locating the computer away from other equipment allows air to flow in and out of the computer. Placement close to other equipment can constrict air flow and increase heat.
 * Fan Air-Flow Direction - Notice the direction the computer case's fans blow air. You want cool air coming in and hot air to exhaust out. Keeping cool air coming in helps control the temperature rating inside the case, which may assist in the life span of the hardware.
 * Cable Management - Having a rat's nest of cables can disrupt airflow and make your computer run hotter. It is recommended to take some time to cable manage after building a computer, as it can increase the lifespan of your computer in the future. There are various accessories that can help with cable management. Some examples are zip ties, velcro ties and cable sleeves.
 
 ## Thermal Paste
-Thermal paste is a thermally conductive compound which is used as an interface between the CPU and the heatsink itself. Thermal paste trys to eliminate the amount of air gapes or space inbetween the heatsink and CPU in order to maxamize the amount of heat transfer in between the two.
+Thermal paste is a thermally conductive compound which is used as an interface between the CPU and the heatsink itself. Thermal paste trying to eliminate the number of air gaps or space between the heatsink and CPU in order to maximize the amount of heat transfer between the two.
```

Comment:
"Thermal paste _tries_ to eliminate..."
To be honest, "Thermal paste aims to eliminate..." sounds better.